ruby/vampires.rb

```diff
@@ -1,30 +1,63 @@
-puts "Enter First Name"
-firstName = gets.chomp
-puts "Enter Middle Initial"
-middleInitial = gets.chomp
-puts "Enter Last Name"
-lastName = gets.chomp
-puts "Enter Age"
-applicantAge = gets.chomp
-puts "Enter Year Born"
-birthYear = gets.chomp
-puts "Would you like some Garlic Bread fresh from our cafeteria?"
-garlicReply = gets.chomp
-puts "Would you like to enroll in the health insurance plan offered by the company?"
-insuranceReply = gets.chomp
-currentYear = Time.new.year
-birthYear - currentYear = ageCheck
-firstName + lastName = fullName
+puts "Welcome to the Werewolf Resources Department"
+puts "How many applicants are being processed at this time?"
+numApplicants = gets.chomp.to_i
```

Comment:
Generally in Ruby we reserve Camel Case for classes, modules, and other constants. Methods and variables should be using underscores. Check out the airbnb Ruby style guide for examples! https://github.com/airbnb/ruby#naming